### ISSUE TITLE
[HW1] About Typescript Assignment

### ISSUE THREAD

**Issue by 93lemondrops** [swsnu/swppfall2017#19]
1. If the age (or birth-day, birth-year...) form is empty, should I think it as 0? Or should I print a pop-up message to correct it (that input should be a number)?

2. Can I make an X icon with just a character X? Or should I make an icon with an image or something?

**Comment by wonook** [comment]
1. It can be handled any way, but if the input is wrong (either 0 or 'empty'), just a simple message to correct the form (specifying which form) would be fine.
2. Just a character X is fine.
3. Yes, in our case, it should exactly match one of given strings of months. "January", "February", ... "December".

**Comment by ktaebum** [comment]
@wonook I am little confused after reading this issue.
    
 So you mean that it's okay to show a simple message (tooltip) with the requirement about certain form when the empty string is input (So,  just handle the empty string input as a simple error format, not as the special case.), except for the 'Age'?? 
 In other words, about the 'Age', after getting the empty string, when I hover 'X' char, it should show kind of message like "It is empty", not the "Must be a number between 0 and 200 (inclusive)". Right?

**Comment by wonook** [comment]
@ktaebum Nope, the 'Age' form doesn't have to be that way. It would be fine as long as it alerts the user that something is wrong with the form. 'Age' form is not an exception. I've mentioned the 'Age' form, as it includes 0 as its 'correct' result, unlike others. It doesn't have to be handled differently. The goal of the homework is for you to have a taste of TypeScript/JavaScript.
